docs/backup-restore.md

```diff
@@ -1,134 +1,183 @@
+## Create a backup
 
+How you create a backup depends on which database you're using and whether or not you're using multiple databases. The following decision tree shows which approach you should take.
 
-The following is the decision tree to decide what approach you should take.
+```mermaid
+flowchart TD
+  A[Are you using a single database with ScalarDB?]
+  A -->|Yes| B[Does the database have transaction support?]
+  B --> C[Perform <a href='#back-up-without-explicit-pausing'>back up without explicit pausing</a>]
+  A ---->|No| E[Perform <a href='#back-up-with-explicit-pausing'>back up with explicit pausing</a>]
+```
 
-- :question: Do you use a single DB under ScalarDB?
-  - :question: Does the DB have transaction support?
-    - :arrow_right: use [backup without explicit pausing](#backup-without-explicit-pausing)
-- Otherwise
-  - :arrow_right: use [backup with explicit pausing](#backup-with-explicit-pausing)
+### Back up without explicit pausing
 
-### Backup without explicit pausing
+If you're using ScalarDB with a single database with support for transactions, you can create a backup of the database even while ScalarDB continues to accept transactions.
 
-If you use ScalarDB on a single DB with transaction support, you can create backups for the DB even while ScalarDB continues to accept transactions.
+One requirement for creating a backup in ScalarDB is that backups for all the ScalarDB-managed tables (including the Coordinator table) need to be transactionally consistent or automatically recoverable to a transactionally consistent state. That means that you need to create a consistent snapshot by dumping all tables in a single transaction.
 
-One requirement for backup in ScalarDB is that backups for all the ScalarDB managed tables (including the coordinator tables) need to be transactionally-consistent or automatically recoverable to a transactionally-consistent state.
-That means that you need to create a consistent snapshot by dumping all tables in a single transaction.
+How to create transactionally consistent backups depends on the type of database. Select your database to see examples for how to create a transactionally consistent backup for ScalarDB.
 
-The ways to create transactional backups are different among each database product.
-We show examples for some database products, but you are requested to find safe ways for a transactional backup from your databases at your own risk.
+{% capture notice--warning %}
+**Attention**
 
-#### MySQL (backup)
+Before creating a backup, you should consider the safest way to create a transactionally consistent backup of your databases and understand any risks that are associated with the backup process.
+{% endcapture %}
 
-Use the `mysqldump` command with `--single-transaction` option.
+<div class="notice--warning">{{ notice--warning | markdownify }}</div>
 
-#### PostgreSQL (backup)
+<div id="tabset-1">
```

Comment:
This doesn't cover all the databases we support. So I think we should note that these are merely examples for some of our supported databases.

Reply:
Understood. I've added a note to this part.